contracts/aggregator/fantom/flashloan/implAaveV3/main.sol

```diff
@@ -0,0 +1,105 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
@shriyatyagii I think folder structure could be something like this:
|-- implementations
     | --  aaveV3
     | --- fla

Reply:
Done!